cartography/intel/aws/inspector.py

```diff
@@ -244,20 +238,35 @@ def load_inspector_findings(
 def load_inspector_packages(
     neo4j_session: neo4j.Session,
     packages: List[Dict[str, Any]],
-    region: str,
```

Comment:
Adding region here adds no value, since the same package-version will probably be installed in multiple instances/containers/lambdas from different regions.
The only important piece is the `AWS_ACCOUNT` where the package was located to delete the package when the account is no longer synced.